openedx/features/content_type_gating/helpers.py

```diff
@@ -0,0 +1,30 @@
+"""
```

Comment:
I wonder if it makes sense to create: "openedx/features/feature_based_enrollments/" and then populate that with various modules that would be shared across the other features.

I could imagine this function going in something like openedx/features/feature_based_enrollments/access.py or something (for example).

Not a blocking request -- we could definitely do something like this later.

Reply:
i agree, this was @cpennington suggestion too, and I think we should put most of the shared code in these functions there.  
but was trying to not do all the refactoring for this PR per your suggestion

Reply:
Make sense!